galaxy/api/internal/urls.py

```diff
@@ -17,9 +17,15 @@
 
 # from django.urls import path
 
-# from galaxy.api.internal import views
+from galaxy.api.internal import views
+from django.conf.urls import url
```

Comment:
Django regexp urls are about to be deprecated. Please use `from django.urls import path`